pennylane/templates/subroutines/hilbert_schmidt.py

```diff
@@ -248,41 +303,49 @@ class LocalHilbertSchmidt(HilbertSchmidt):
 
         .. code-block:: python
 
-            import numpy as np
+            U = qml.CZ(wires=(0,1))
 
-            with qml.QueuingManager.stop_recording():
-                u_tape = qml.tape.QuantumTape([qml.CZ(wires=(0,1))])
-
-            def v_function(params):
-                qml.RZ(params[0], wires=2)
-                qml.RZ(params[1], wires=3)
-                qml.CNOT(wires=[2, 3])
-                qml.RZ(params[2], wires=3)
-                qml.CNOT(wires=[2, 3])
+            def V_function(params):
+                return [qml.RZ(params[0], wires=2),
+                        qml.RZ(params[1], wires=3),
+                        qml.CNOT(wires=[2, 3]),
+                        qml.RZ(params[2], wires=3),
+                        qml.CNOT(wires=[2, 3])]
```

Comment:
```suggestion
            V = [qml.RZ(params[0], wires=2),
                    qml.RZ(params[1], wires=3),
                    qml.CNOT(wires=[2, 3]),
                    qml.RZ(params[2], wires=3),
                    qml.CNOT(wires=[2, 3])
             ]
```
not exactly sure on the indentation though.